manifest/docker_schema2.go

```diff
@@ -78,7 +78,7 @@ type Schema2Config struct {
 	Tty             bool                 // Attach standard streams to a tty, including stdin if it is not closed.
 	OpenStdin       bool                 // Open stdin
 	StdinOnce       bool                 // If true, close stdin after the 1 attached client disconnects.
-	Env             []string             // List of environment variable to set in the container
+	Env             []string             `json:",omitempty"` // List of environment variable to set in the container
```

Comment:
Why  the “omit empty” marking? It is not used in docker/docker, and only applies to encoding, i.e. should not be needed for inspecting at all.